### ISSUE TITLE
Should CA1200 nag when cref array?

### ISSUE THREAD

**Issue by JohanLarsson** [dotnet/roslyn-analyzers#2004]
[CA1200](https://github.com/dotnet/roslyn-analyzers/blob/32d8f1e397439035f0ecb5f61a9e672225f0ecdb/src/Microsoft.CodeQuality.Analyzers/Core/Documentation/AvoidUsingCrefTagsWithAPrefix.cs) nags here:

```cs
/// <summary>
/// <see cref="T:int[]"/>
/// </summary>
public void M()
{
}
```

CS1584 nags here:

```cs
/// <summary>
/// <see cref="int[]"/>
/// </summary>
public void M()
{
}
```

**Comment by sharwell** [comment]
What is CA1200? FWIW, neither of those are valid references. You cannot directly `cref` an array type.

**Comment by mavasani** [comment]
CA1200 is a new rule that was initially written for Roslyn.sln but was promoted to generic code quality rule. https://github.com/dotnet/roslyn-analyzers/blob/32d8f1e397439035f0ecb5f61a9e672225f0ecdb/src/Microsoft.CodeQuality.Analyzers/Core/Documentation/AvoidUsingCrefTagsWithAPrefix.cs
